locksmith/__tests__/controllers/userController.test.ts

```diff
@@ -336,7 +341,12 @@ describe('User Controller', () => {
           .put('/users/user@example.com/paymentdetails')
           .set('Accept', /json/)
           .send({
-            token: 'an_invalid_token',
+            message: {
+              user: {
+                publicKey: '0xAaAdEED4c0B861cB36f4cE006a9C90BA2E43fdc2',
+                stripeTokenId: 'tok_visa',
```

Comment:
as a fast follow it might be worthwhile of us to change the token or something to indicate that the payload is part of a failure scenario

Reply:
Good idea, I’ll do that before I merge